apps/condo/domains/common/utils/GraphQLLoggerApp.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const gql = require('graphql-tag')
+const { get, set } = require('lodash')
+
+function normalizeQuery (string) {
+    if (!string) return ''
+    return string.replace(/[\s,]+/g, ' ').trim()
+}
+
+function normalizeVariables (object) {
+    if (!object) return undefined
+    const data = JSON.parse(JSON.stringify(object))
+    for (const key of ['secret', 'password', 'data.password', 'data.secret']) {
```

Comment:
this array will be recreated on every execution of the function, better to move to constant outside of function